client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -261,16 +277,40 @@ const Organization = ({ organizationId }: { organizationId: string }) => {
         <Column>
           <H3>Audits</H3>
           <ButtonList>
-            {elections.map(election => (
-              <LinkButton
-                key={election.id}
-                to={`/support/audits/${election.id}`}
-                intent={Intent.PRIMARY}
-              >
-                {election.auditName}
-              </LinkButton>
-            ))}
+            {elections
+              .filter(election => !election.deletedAt)
+              .map(election => (
+                <LinkButton
+                  key={election.id}
+                  to={`/support/audits/${election.id}`}
+                  intent={Intent.PRIMARY}
+                >
+                  {election.auditName}
+                </LinkButton>
+              ))}
           </ButtonList>
+          <H3>Deleted Audits</H3>
+          <Table striped>
+            <tbody>
+              {elections
+                .filter(election => !election.deletedAt)
```

Comment:
```suggestion
                .filter(election => election.deletedAt)
```

shouldn't it be this ^^ ?

Reply:
whoops, fixed and covered with a test